<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COM_port</class>
 <widget class="QWidget" name="COM_port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bluetooth_Connect</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>200</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>COM_pot_text.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>95</y>
     <width>260</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:flat {   border: none; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>connect_button.png</normaloff>connect_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1000</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="portline_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>23</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>COM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
